### PR TITLE
[i18n] Improve localization  file sanitizer

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Improve localization  file sanitizer

### CODE REVIEW

File: packages/strapi-plugin-i18n/services/core-api.js [hunk 1]
```diff
@@ -22,25 +22,85 @@ const parseRequest = ctx => {
...
+      },
+      {},
+      keys(files)
+    );
   };
 
-  const getAllLocalizations = entry => {
-    return [entry.id, ...map(prop('id'), entry.localizations)];
+  /**
+   * Sanitizes uplaoded files to keep only writable ones
```
Comment (petersg83) [inline packages/strapi-plugin-i18n/services/core-api.js:83]: uploaded*

Comment (petersg83) [reply packages/strapi-plugin-i18n/services/core-api.js:83]: and bad copy paste

**Review by petersg83 — CHANGES_REQUESTED** [review]
LGTM, few typos.
The sanitize works for the first level, should it also work for other levels? (components)

**Comment by alexandrebodin** [comment]
> LGTM, few typos.
> The sanitize works for the first level, should it also work for other levels? (components)

If a component is localized its fields are allowed  so no need

**Review by petersg83 — APPROVED** [review]
LGTM :)

**Pull request merged**